docs/docs/50-user-guide/60-reference-docs/90-events/100-notifications/10-configuring-routers.md

```diff
@@ -0,0 +1,152 @@
+---
+description: How to configure and use EventRouters for notifications
+---
+
+# Configuring Event Routers
+
+<span class="tag professional"></span>
+<span class="tag beta"></span>
+
+[Event Routers](./00-overview.md#event-routers) are used to route events to different destinations
+based on specified criteria. This document provides guidance on how to configure and use Event
+Routers in Kargo.
+
+:::info
+
+Before creating Event Routers, ensure that you or your administrator have set up the necessary
+[`ClusterMessageChannel`](../../../../40-operator-guide/35-cluster-configuration.md#cluster-message-channels)
+or [`MessageChannel`](../../../20-how-to-guides/20-working-with-projects.md#message-channels)
+resources that define the destinations for your notifications.
+
+:::
+
+## Managing Event Routers
+
+You can manage Event Routers through the Kargo UI or declaratively via YAML configuration files
+deployed by Argo CD. To manage it in the UI, go to your Project settings, then in the sidebar,
+select "Notifications." In this section you can create, update, and delete both your channels and
+your event routers.
+
+:::note
+
+Currently, event routers can only be created by editing YAML, which you can do in the UI. We plan to
+improve this experience in future releases.
+
+:::
+
+The simplest form of an event router configuration looks like this:
+
+```yaml
+kind: EventRouter
+apiVersion: ee.kargo.akuity.io/v1alpha1
+metadata:
+  name: promotion-status
+  namespace: kargo-demo
+spec:
+  types:
+    - PromotionSucceeded
+    - PromotionFailed
+    - PromotionErrored
+  channels:
+    - name: slack
+      kind: MessageChannel
+```
+
+In this example, the event router named `test-router` is configured to listen for two [event
+types](../10-event-reference.md#event-types): `PromotionCreated` and `PromotionErrored`. When either
+of these events occurs, a notification will be sent to the channel named `slack`, which is of kind
+`MessageChannel` using a default message template defined for the type of event along with the
+channel type. You can listen for multiple event types and send notifications to multiple channels by
+adding more entries to the `types` and `channels` lists.
+
+:::warning
+
+If you're using default message templates, the specified event types _must_ all be of the same
+"class" (such as Promotion events, Freight events, etc.). Mixing event types from different classes
+is not supported when using default templates and will error.
```

Comment:
Does this error when routing? Or is it a validation error? Just asking.

Reply:
Right now it is when it tries to route. We can add this to the webhook we'll be releasing though (already have it captured if I remember correctly)